### ISSUE TITLE
Some 7tv colors showing shadow instead of glow

### ISSUE THREAD

**Issue by Baleur** [SevenTV/chatterino7#276]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Some random 7tv name colors dont show glow but shows a dropshadow instead.
Most name colors display glow correctly however.
Correct glow in first screenshot from browser, second screenshot from latest stable chatterino 7tv fork.
The one in the screenshot is "Rosé"

Confirmed its still an issue in latest nightly build as of Oct 03

### Screenshots

<img width="208" alt="image" src="https://github.com/user-attachments/assets/1f6ca121-4207-4920-b070-86fc2d34a734">

<img width="171" alt="image" src="https://github.com/user-attachments/assets/3c616693-9977-4533-9eae-5d7f23cc8577">

### OS and Chatterino Version

Chatterino 7.5.1 (commit b281c60e) built with Qt 6.5.0, MSVC 193833135 Running on Windows 11 Version 23H2, kernel: 10.0.22631

**Comment by Nerixyz** [comment]
> Some random 7tv name colors dont show glow but shows a dropshadow instead.

There are no glows in 7TV paints - only drop shadows.

---

This is more of an issue with Qt's pixmap filters not working the same as CSS filters, I think.

**Comment by Baleur** [comment]
Yeah sorry thats what i meant, effectively the shadow "looks like" glow
